zhaquirks/sinope/light.py

```diff
@@ -23,79 +27,194 @@
+            id=0x0004, type=t.CharacterString, access="r", is_manufacturer_specific=True
+        )
+        on_intensity: Final = foundation.ZCLAttributeDef(
+            id=0x0010, type=t.int16s, access="rw", is_manufacturer_specific=True
+        )
+        on_led_color: Final = foundation.ZCLAttributeDef(
+            id=0x0050, type=t.uint24_t, access="rw", is_manufacturer_specific=True
+        )
+        off_led_color: Final = foundation.ZCLAttributeDef(
+            id=0x0051, type=t.uint24_t, access="rw", is_manufacturer_specific=True
+        )
+        on_led_intensity: Final = foundation.ZCLAttributeDef(
+            id=0x0052, type=t.uint8_t, access="rw", is_manufacturer_specific=True
+        )
+        off_led_intensity: Final = foundation.ZCLAttributeDef(
+            id=0x0053, type=t.uint8_t, access="rw", is_manufacturer_specific=True
+        )
+        action_report: Final = foundation.ZCLAttributeDef(
+            id=0x0054, type=ButtonAction, access="rp", is_manufacturer_specific=True
+        )
+        min_intensity: Final = foundation.ZCLAttributeDef(
+            id=0x0055, type=t.uint16_t, access="rw", is_manufacturer_specific=True
+        )
+        phase_control: Final = foundation.ZCLAttributeDef(
+            id=0x0056, type=PhaseControl, access="rw", is_manufacturer_specific=True
+        )
+        double_up_full: Final = foundation.ZCLAttributeDef(
+            id=0x0058, type=DoubleFull, access="rw", is_manufacturer_specific=True
+        )
+        current_summation_delivered: Final = foundation.ZCLAttributeDef(
+            id=0x0090, type=t.uint32_t, access="rp", is_manufacturer_specific=True
+        )
+        timer: Final = foundation.ZCLAttributeDef(
+            id=0x00A0, type=t.uint32_t, access="rw", is_manufacturer_specific=True
+        )
+        timer_countdown: Final = foundation.ZCLAttributeDef(
+            id=0x00A1, type=t.uint32_t, access="r", is_manufacturer_specific=True
+        )
+        connected_load: Final = foundation.ZCLAttributeDef(
+            id=0x0119, type=t.uint16_t, access="rw", is_manufacturer_specific=True
+        )
+        status: Final = foundation.ZCLAttributeDef(
+            id=0x0200, type=t.bitmap32, access="rp", is_manufacturer_specific=True
+        )
+        cluster_revision: Final = foundation.ZCL_CLUSTER_REVISION_ATTR
+
+    server_commands = {
+        0x54: foundation.ZCLCommandDef(
+            "button_press",
+            {"command": t.uint8_t},
+            direction=foundation.Direction.Server_to_Client,
+            is_manufacturer_specific=True,
+        )
     }
 
+    def handle_cluster_general_request(
+        self,
+        hdr: foundation.ZCLHeader,
+        args: list[Any],
+        *,
+        dst_addressing: Optional[
+            Union[t.Addressing.Group, t.Addressing.IEEE, t.Addressing.NWK]
+        ] = None,
+    ):
+        """Handle the cluster command."""
+        self.debug(
```

Comment:
We can leave it like this for now, but this causes a `zigpy` log. We might want to use our own logger here, since that'll clearly be `zhaquirks.sinope`. Other quirks aren't consistent about this though.
The string does include "Sinope" at least, so that's good.